pooch/tests/data/registry-spaces.txt

```diff
@@ -0,0 +1,2 @@
+"tiny space data.txt" baee0894dba14b12085eacb204284b97e362f4f3e5a5807693cc90ef415c1b2d
+tiny\ space\ data.txt baee0894dba14b12085eacb204284b97e362f4f3e5a5807693cc90ef415c1b2d
```

Comment:
Maybe make this a different file name so we can check for its existence in the test? As it is, this will clobber the entry above.